SygicMapsKit/Classes/BrowseMap/SYMKBrowseMapViewController.swift

```diff
@@ -37,9 +64,24 @@ public class SYMKBrowseMapViewController: UIViewController {
     private func setupViewDelegates() {
         guard let view = view as? SYMKBrowserMapView, let mapView = view.mapView else { return }
         mapView.delegate = self
+        view.recenter.addTarget(self, action: #selector(SYMKBrowseMapViewController.didTapRecenterButton), for: .touchUpInside)
         view.compass.delegate = self
     }
     
+    private func setupMapSelectionManager() {
+        guard let view = view as? SYMKBrowserMapView, let mapView = view.mapView else { return }
+        mapSelectionManager.mapObjectsManager = self
+        mapSelectionManager.clusterLayer = SYMapMarkersCluster()
+        mapView.addMapMarkersCluster(self.mapSelectionManager.clusterLayer!)
+    }
+    
+    // MARK: - Actions
+    
+    @objc func didTapRecenterButton() {
+        guard let view = view as? SYMKBrowserMapView, let mapView = view.mapView else { return }
+        mapView.cameraMovementMode = .followGpsPositionWithAutozoom
+        mapView.cameraRotationMode = .vehicle
```

Comment:
toto chceme vzdy nastavovat na vehicle? nechceme tam default lock na sever resp az viacnasobne tapanie na button by prepinalo medzi lock na sever a .vehicle?

Reply:
neviem, nemame definovane ako sa to ma spravat, tak som tam zatial dal len taky zaklad.